e2e/cypress/integration/search.spec.js

```diff
@@ -1,5 +1,5 @@
 describe("Search", () => {
-  context(`Form`, () => {
```

Comment:
the context is something for the mockups - ask @kfudbc

Reply:
context og describe gør vist det samme